### PR TITLE
Update communication.json

### PR DESCRIPTION
**Pull request by deksmond**
I updated this page with the list of Senators(not complete)
N.A means Not Available
The 3 South-West Lagos are not errors, that's how I found the info

### CODE REVIEW

File: src/main/resources/communication.json [hunk 1]
```diff
@@ -1,10 +1,169 @@
...
+  {
+    "governmentOfficialName": "SEN. ADAEZE ODUAH",
+    "governmentPosition": "Senator South-East Anambra",
+    "phoneNumber": "08055084340",
+    "phoneMessage": "#EndSarsNow",
+    "email": "senatorstella@gmail.com",
+    "emailMessage": "#EndSarsNow"
+  },
+  
+    "governmentOfficialName": "SEN. ADAMU BULKACHUWA",
```
Comment (oyekanmiayo) [inline src/main/resources/communication.json:75]: Hey, there are some errors here. Pls can you fix?

Comment (deksmond) [reply src/main/resources/communication.json:75]: Okay. On it

**Comment by oyekanmiayo** [comment]
@deksmond I've merged a PR that changed the file you've created a PR for. Pls, check that there are no conflicts and repetitions. Thanks.

**Comment by deksmond** [comment]
Just checked. Changes are perfect. Thanks

**Pull request closed without merging**